bin/tsc.js

```diff
@@ -21,6 +21,12 @@ var ts;
         ExitStatus[ExitStatus["DiagnosticsPresent_OutputsGenerated"] = 2] = "DiagnosticsPresent_OutputsGenerated";

```

Comment:
ok now i see. u r using it later on. still i would do that in different changes.